subsys/net/l2/ethernet/gptp/gptp_state.h

```diff
@@ -159,16 +159,18 @@ struct gptp_priority_vector {
 	/** Identity of the source clock. */
 	struct gptp_root_system_identity root_system_id;
 
-	/** portNumber of the receiving port. */
-	uint16_t port_number;
+	/** Steps removed from the announce message transmitter and the
+	 * master clock. Note that this field must be right after
+	 * root_system_id as we are comparing root system id and steps
+	 * removed in one memcmp()
+	 */
+	uint16_t steps_removed;
 
 	/** Port identity of the transmitting time-aware system. */
 	struct gptp_port_identity src_port_id;
 
-	/** Steps removed from the announce message transmitter and the
-	 * master clock.
-	 */
-	uint16_t steps_removed;
+	/** portNumber of the receiving port. */
```

Comment:
Given this is already modified, shouldn't this be `Port number`?

Reply:
Well, the variable is called portNumber in the spec so the value refers to that. On the other hand, we do not have similar naming for stepsRemoved. I can change the comment.